docs/ssh.md

```diff
@@ -0,0 +1,28 @@
+# SSH Configuration
```

Comment:
does the main readme link here at all? i feel like this should be known early on otherwise setup won't work right?

Reply:
Yep, it's linked to from the main readme